<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Disass</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/Artwork/Icons/Logo.png</normaloff>:/Artwork/Icons/Logo.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sendButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Send command to GDB</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/Controls/Icons/Controls/Send_Button.png</normaloff>:/Controls/Icons/Controls/Send_Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>42</width>
      <height>44</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="pidBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>PID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>550</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>(Re)start the program</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/Controls/Icons/Controls/Start_Button.png</normaloff>:/Controls/Icons/Controls/Start_Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>29</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Command</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="atNtButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Set syntax to AT&amp;T</string>
    </property>
    <property name="text">
     <string notr="true">AT&amp;&amp;T Syntax</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="intelButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Set syntax to Intel</string>
    </property>
    <property name="text">
     <string>Intel Syntax</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="codeOutputTabs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>591</width>
      <height>461</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="assemblyOutput">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Disassembly</string>
     </attribute>
     <widget class="QTextBrowser" name="gdbAsmOutputBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="codeOutput">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Code</string>
     </attribute>
     <widget class="QTextBrowser" name="gdbCodeOutputBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gdbOutput">
     <attribute name="title">
      <string>GDB Output</string>
     </attribute>
     <widget class="QTextBrowser" name="gdbOutputBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="breakButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Break</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/Controls/Icons/Controls/Break_Button.png</normaloff>:/Controls/Icons/Controls/Break_Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="breakBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Address/Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepIntoButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Step Into (next/nexti)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/Controls/Icons/Controls/Step_Into_Button.png</normaloff>:/Controls/Icons/Controls/Step_Into_Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stepOverButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Step Over (step/stepi)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/Controls/Icons/Controls/Step_Over_Button.png</normaloff>:/Controls/Icons/Controls/Step_Over_Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Continue running (continue)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/Controls/Icons/Controls/Continue_Button.png</normaloff>:/Controls/Icons/Controls/Continue_Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>26</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nextCodeLineButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Next Code Line (until)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/Controls/Icons/Controls/Next_Line_Button.png</normaloff>:/Controls/Icons/Controls/Next_Line_Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>27</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Stop Debugging</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Icons.qrc">
      <normaloff>:/Controls/Icons/Controls/Stop_Button.png</normaloff>:/Controls/Icons/Controls/Stop_Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>26</width>
      <height>26</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandLineArgumentsBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>550</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Program Arguments</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="registerBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>341</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="stackBox">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>141</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="fileArchBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="cursorWidth">
     <number>1</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
    <property name="placeholderText">
     <string>File Architecture</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="fileNameBox">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="cursorWidth">
     <number>1</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
    <property name="placeholderText">
     <string>Program Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_Use"/>
   </widget>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>Configure</string>
    </property>
    <widget class="QMenu" name="menuForce_Architecture">
     <property name="title">
      <string>Force Architecture</string>
     </property>
     <addaction name="action64_Bit"/>
     <addaction name="action32_Bit"/>
    </widget>
    <addaction name="menuForce_Architecture"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
    <addaction name="actionContribute"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfigure"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Select Executable</string>
   </property>
   <property name="toolTip">
    <string>Select an executable to debug</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Info</string>
   </property>
   <property name="toolTip">
    <string>Info about Disass</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License and Credit</string>
   </property>
  </action>
  <action name="actionContribute">
   <property name="text">
    <string>Contribute</string>
   </property>
   <property name="toolTip">
    <string>Contribute to Disass</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action64_Bit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>x86-64 (64-Bit)</string>
   </property>
   <property name="toolTip">
    <string>Force the architecture to be x86-64 (64-Bit)</string>
   </property>
  </action>
  <action name="action32_Bit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>x86 (32-Bit)</string>
   </property>
   <property name="toolTip">
    <string>Force the architecture to be x86 (32-Bit)</string>
   </property>
  </action>
  <action name="actionHow_to_Use">
   <property name="text">
    <string>How to Use</string>
   </property>
   <property name="toolTip">
    <string>Tutorial on how to use Disass</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
